rest/rest-server/src/test/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRestJobLogsTest.java

```diff
@@ -105,9 +107,11 @@ public void job_full_logs_not_finished() throws Exception {
         jobState.addTask(new InternalScriptTask(jobState));
         when(mockScheduler.getJobState("123")).thenReturn(jobState);
 
-        InputStream fullLogs = restScheduler.jobFullLogs(validSessionId, "123", validSessionId);
+        InputStream fullLogs = (InputStream) restScheduler.jobFullLogs(validSessionId, "123", validSessionId, null)
```

Comment:
@fviale  Wouldn't it be useful to add a test alternating the sessionId parameter (one in the header & one in the query params) ?

Reply:
ok

Reply:
I don't think this test would be useful